man/sd_bus_set_method_call_timeout.xml

```diff
@@ -97,7 +97,6 @@
       <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>sd-bus</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>sd_bus_call</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>sd_bus_message_seal</refentrytitle><manvolnum>3</manvolnum></citerefentry>
```

Comment:
Hmm, now we have a trailing comma ;( But let's fix that later.

Reply:
Woops, I'll fix that in the next PR.